daemon.go

```diff
@@ -146,3 +147,41 @@ func (d *Daemon) listen() {
 		go d.handleConn(c)
 	}
 }
+
+func clearUnixSockets(maddr ma.Multiaddr) error {
+	for _, subMaddr := range ma.Split(maddr) {
+		if subMaddr.Protocols()[0].Code == ma.P_UNIX {
```

Comment:
Prefer a negative check here and `continue` to avoid right-shifting.

Reply:
Done